client/src/stores/configurationStore.ts

```diff
@@ -17,11 +15,14 @@ export const useConfigStore = defineStore("configurationStore", () => {
         if (!isLoaded.value && !isLoading.value) {
             isLoading.value = true;
             try {
-                const { data } = await fetchConfiguration({});
+                const { data, error } = await GalaxyApi().GET("/api/configuration");
+
+                if (error) {
+                    console.error("Error loading Galaxy configuration", error);
```

Comment:
😢 In a follow up I would say we should add another property to the store that holds the error.